src/Factory/KeyPairFactory.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the ACME PHP library.
+ *
+ * (c) Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AcmePhp\Ssl\Factory;
+
+use AcmePhp\Ssl\KeyPair;
+use AcmePhp\Ssl\PrivateKey;
+use AcmePhp\Ssl\PublicKey;
+
+/**
+ * Crate KeyPair entities from PEMs.
+ *
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+class KeyPairFactory
```

Comment:
I don't like this kind of factories. Either we use directly the KeyPair or we create an interface that the factory implements. The aim of the Factory pattern is to be used with Strategy so anyone can replace it implementing the interface themselves. IMO without an interface it's pointless.